.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,3 +0,0 @@
-Thanks for contributing to DLR! By submitting this pull request, you confirm that your contribution is made under the terms of the [Apache 2.0 license](https://github.com/neo-ai/neo-ai-dlr/blob/master/LICENSE).
```

Comment:
Why is this removed?

Reply:
This seems a hidden file of Github while creating PR. I can't see such a file in other PR.
Shall I keep it in the PR ?

Reply:
It's github's PR template as suggested by [this link](https://help.github.com/en/github/building-a-strong-community/creating-a-pull-request-template-for-your-repository). Please keep it.

Reply:
Done